src/main/java/com/Acrobot/ChestShop/Commands/ItemInfo.java

```diff
@@ -39,31 +39,10 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
             Bukkit.getPluginManager().callEvent(parseEvent);
             item = parseEvent.getItem();
         }
-
-        if (MaterialUtil.isEmpty(item)) {
-            return false;
-        }
-
-        iteminfo.send(sender);
-        try {
-            iteminfo_fullname.send(sender, "item", MaterialUtil.getName(item));
-        } catch (IllegalArgumentException e) {
-            sender.sendMessage(ChatColor.RED + "Error while generating full name. Please contact an admin or take a look at the console/log!");
-            ChestShop.getPlugin().getLogger().log(Level.SEVERE, "Error while generating full item name", e);
-            return true;
-        }
-
-        try {
-            iteminfo_shopname.send(sender, "item", MaterialUtil.getSignName(item));
-        } catch (IllegalArgumentException e) {
-            sender.sendMessage(ChatColor.RED + "Error while generating shop sign name. Please contact an admin or take a look at the console/log!");
-            ChestShop.getPlugin().getLogger().log(Level.SEVERE, "Error while generating shop sign item name", e);
-            return true;
-        }
-
+    
         ItemInfoEvent event = new ItemInfoEvent(sender, item);
         ChestShop.callEvent(event);
-
-        return true;
+    
+        return !event.isCancelled() && !MaterialUtil.isEmpty(item);
```

Comment:
You don't want to return false here on event cancel, that would show the command's usage message. I would just leave the original empty check in there or add them to all the other listeners so that they don't error.